__tests__/components/fields/object.js

```diff
@@ -136,6 +137,8 @@ describe('object field', () => {
       ObjectClass
     );
 
+    delete object.state.id;
```

Comment:
What if you just change the test to use `object.state.assocList` since that's all it uses from the state anyway?